Cargo.toml

```diff
@@ -9,7 +9,7 @@ rust-version = "1.71"
 [dependencies]
 serde = { version = ">=1.0", features = ["derive"] }
 chrono = "0.4"
-clap = "3.2"
+clap = "~4.4"
```

Comment:
We should consider to refactor this to use the clap derive feature in the future, might simplify the code a little.